stator/models.py

```diff
@@ -44,8 +44,11 @@ def add_stator_indexes(sender, **kwargs):
     Inject Indexes used by StatorModel in to any subclasses. This sidesteps the
     current Django inability to inherit indexes when the Model subclass defines
     its own indexes.
+
+    We should only add this for models that direct descendants of
+    StatorModel, otherwise we will see system check errors.
     """
-    if issubclass(sender, StatorModel):
+    if sender.__base__ is StatorModel:
```

Comment:
I'm not completely thrilled with this--`__base__` is semi-documented, and points to the first parent class.

But I'm not sure what multiple inheritance would do in the context of Django models.